spectrum_io/search_result/sage.py

```diff
@@ -14,7 +14,7 @@
 class Sage(SearchResults):
     """Handle search results from Sage."""
 
-    def read_result(self, tmt_labeled: str = "") -> pd.DataFrame:
+    def read_result(self, tmt_labeled: str = "", custom_mods: Dict[str, str] = None) -> pd.DataFrame:
```

Comment:
wrong type nnotation, this is Optional[Dict[str, str]], because it can be also None.

Reply:
also, this function is inherited from the superclass function SearchResults::read_result, which has a different signature, because it expects two additional inputs, separated into static and var mods. The signature and type annotations must be exactly the same!